src/components/Global/GlobalInformation.js

```diff
@@ -0,0 +1,21 @@
+import React from "react"
+
+import TotalValue from "./TotalValue"
+import data from "../../data/stats.json"
+import "./Global.css"
+
+const GlobalInformation = () => {
+  return (
+    <section className="global">
```

Comment:
Imho an [article tag](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/article) would fit better here as described in [section tag](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section) documentation (There is a little yellow note about it) 😄 

There is another section tag in `Landing.js`